### ISSUE TITLE
Is F_c added with F_enc better than just montage them?

### ISSUE THREAD

**Issue by Psw-coder** [TencentARC/T2I-Adapter#121]
When I montage them,I just get this image.Do you make the same experiments?
![166](https://github.com/TencentARC/T2I-Adapter/assets/108187964/8d7cdcc7-7af2-4c01-adbe-20da2d0ec957)

![199](https://github.com/TencentARC/T2I-Adapter/assets/108187964/7fcfc71f-0b8d-4f29-9410-7cf33b3303e6)
![117](https://github.com/TencentARC/T2I-Adapter/assets/108187964/7c60745b-b3b2-4733-9b77-6f34afae72f1)

**Comment by hda-xian** [comment]
yes,I meet the same problem. I have a question that why are the UNet parameters frozen during training for SD1v5, but not for SDXL? the huggingface training script sets  "Unet.train()" ???
